06_System_Hacking.md

```diff
@@ -48,18 +48,18 @@ Example:
 
 #### Active Online Attack
 
-Directly interact with the target for cracking password.
+Directly interact with the target for cracking the password.
 
 ##### Dictionary Attack
 
-A password cracking application is used along with with a dictionary file. 
-This dictionary file contains entire dictionary or a list of known and common words. 
+A password-cracking application is used along with a dictionary file. 
```

Comment:
Pasword cracking is without dash. See: https://en.wikipedia.org/wiki/Password_cracking